lib/libutee/sub.mk

```diff
@@ -12,5 +12,11 @@ srcs-y += tee_api_objects.c
 srcs-y += tee_api_operations.c
 srcs-y += tee_api_se.c
 srcs-y += tee_api_panic.c
+srcs-y += utf8.c
```

Comment:
Do you think it could be worth to introduce a user-lib specific to TUI instead of having everything in `libutee`? @jbech-linaro @jforissier what do you think?